README.md

```diff
@@ -118,18 +118,30 @@ Flow2ML for Python can be installed from Source,Pypi and Docker container instal
     flow = Flow( 'dataset_dir' , 'data_dir' )
 
     # Define The Filters to be used
-    filters = ["median", "laplacian", "gaussian", "sobelx", "sobely","bilateral"]
+    filters = ["median", "laplacian", "gaussian", "sobelx", "sobely", "bilateral"]
 
     # Apply The Filters
     flow.applyFilters( filters )
 
     # Define The augmentation operations to be used
    
-    operations = {'flip': 'horizontal', 'rotate': 90, 'shear': {'x_axis': 5, 'y_axis': 15}, 'crop': [50, 100, 50, 100], 'scale': 0.1, 'zoom': 2, 'Hist_Equal':False, 'greyscale': True, 'CLAHE':False, 'invert':False, 'erode':False, 'dilate':False, 'open':False, 'close':False,'threshold':{'type':'adaptive','thresh_val':0},'color-space':{'input':'BGR','output':'BGR'}}
+    operations = {'flipped': 'horizontal', 'rotated': 90, 'sheared': {'x_axis': 5, 'y_axis': 15}, 'cropped': [50, 100, 50, 100], 'scaled': 0.1, 'zoomed': 2, 'histogramequalised':False, 'greyscale': True, 'CLAHE':False, 'inverted':False, 'eroded':False, 'dilated':False, 'opened':False, 'closed':False,'thresholded':{'type':adaptive,'thresh_val':0}}
```

Comment:
Why did you remove color-space?

Reply:
Sorry. Actually there was a merge conflict in readme so this line got removed. I'll add it back

Reply:
@rubyruins, can you change the cropping to random crop instead of a list.

Generally we would use random cropping.

It's okay to specify the points, but try to check it with random crop also.

Reply:
@yvkrishna random crop is also implemented in data_augmentation.py. This is just some sample code in the readme.